src/main/webapp/map.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Map Project</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCII0kJqaa9uBysCMxSVcYJn1d2KeuAcDU"></script>
+  <script>
```

Comment:
Even though this script is very small right now, it is better to move it into a separate file, because it will probably grow. This will also make subsequent changes simpler.

Reply:
Moved the style and script to separate files